t/venom/lib/customA.yml

```diff
@@ -0,0 +1,15 @@
+executor: customA
+input:
+  myArg: "world2"
+steps:
+- type: customB
+  myWord: "{{.input.myArg}}"
+  vars:
+    content:
+      from: result.foobarb
```

Comment:
If I understand correctly your examples, the key point is here. 

In order to reuse a variable from executor B into executor A, you have to use `result.name_of_output_variable_in_executor_b` here.

I'm surprised by the syntax because I expected: `result.foobarB`

Reply:
This is the way used in venom, examples.
Let me try this way.

Reply:
Ok it looks that it can work without rename/reassign it.